app/controllers/events_controller.rb

```diff
@@ -0,0 +1,25 @@
+class EventsController < ApplicationController
+  def new; end
+
+  def index
+    @events = Event.all
+  end
+
+  def create
+    @event = current_user.events.build(description: params[:description],
+                                       event_date: params[:event_date],
+                                       location: params[:location])
```

Comment:
Consider using `event_params` pattern as a private method in order to call this as `current_user.events.build(event_params)`

[`OPTIONAL`]